Formula/arpoison.rb

```diff
@@ -21,6 +21,7 @@ class Arpoison < Formula
   depends_on "libnet"
 
   def install
+    on_linux { inreplace "Makefile", /gcc -lnet (.*)/, "gcc \\1 -lnet" }
```

Comment:
We should report this upstream.

Reply:
There doesn't seem to be any publicly visible way of reporting issues. Only option seems to be via email.

We could also just run build command directly as `make` is just:
```
gcc -lnet -o arpoison `libnet-config --defines` -I/opt/local/include -L/opt/local/lib arpoison.c
```

The `-I` and `-L` are no-ops for Homebrew, so it is really just:
```
gcc arpoison.c -o arpoison `libnet-config --defines` -lnet
```
Which is similar to Gentoo's approach https://gitweb.gentoo.org/repo/gentoo.git/tree/net-analyzer/arpoison/arpoison-0.7.ebuild#n26

Reply:
Email is fine. Let's use `patch :DATA` so it's easier to tell when this is no longer needed.